README.md

```diff
@@ -1,80 +1,90 @@
-
-```sh
-make undeploy
-```
-
-## Contributing
-// TODO(user): Add detailed information on how you would like others to contribute to this project
-
-### How it works
-This project aims to follow the Kubernetes [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
-
-It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/) 
-which provides a reconcile function responsible for synchronizing resources untile the desired state is reached on the cluster 
-
-### Test It Out
-1. Install the CRDs into the cluster:
-
-```sh
-make install
-```
-
-2. Run your controller (this will run in the foreground, so switch to a new terminal if you want to leave it running):
-
-```sh
-make run
-```
-
-**NOTE:** You can also run this in one step by running: `make install run`
-
-### Modifying the API definitions
-If you are editing the API definitions, generate the manifests such as CRs or CRDs using:
 
-```sh
-make manifests
-```
 
-**NOTE:** Run `make --help` for more information on all potential `make` targets
+## Prerequisites:
+* AWS
+* Kind 
+    * `make kind`
+* yq 
+    * `make yq`
+
+### 1. Running the operator in the cluster:
+
+
+1. Setup your local environment 
+    ```sh
+    make local_setup MCTC_WORKLOAD_CLUSTERS_COUNT=<NUMBER_WORKLOAD_CLUSTER>
+    ```
+
+1. Update the manager to contain the necessary environment variable and update the image policy:
+    ```
+    containers:
+        - command:
+            - /manager
+            args:
+            - --leader-elect
+            image: controller:latest
+            imagePullPolicy: Never
+            env:
+            - name: AWS_ACCESS_KEY_ID
+            value: <AWS_ACCESS_KEY_ID>
+            - name: AWS_SECRET_ACCESS_KEY
+            value: <AWS_SECRET_ACCESS_KEY>
+            - name: AWS_DNS_PUBLIC_ZONE_ID
+            value: <AWS_DNS_PUBLIC_ZONE_ID>
+            - name: ZONE_ROOT_DOMAIN
+            value: <ZONE_ROOT_DOMAIN>
+    ```
+
+3. Build the image
+    ```sh
+    make docker-build
+    ```
+4. Load the image it into the control plane cluster
+
+    ```sh
+    kind load docker-image controller:latest --name mctc-control-plane  --nodes mctc-control-plane-control-plane
+    ```
+
+5. Deploy the controller to the control plane cluster
+    ```sh
+    make deploy
+    ```
+
+## 2. Running locally:
+1. Create two env files:
+    * One called `aws-credentials.env` containing **AWS_ACCESS_KEY_ID** and **AWS_SECRET_ACCESS_KEY**
+    * One called `controller-config` containing **AWS_DNS_PUBLIC_ZONE_ID** and **ZONE_ROOT_DOMAIN**
+
+
+2.  Setup your local environment 
+
+    ```sh
+    make local_setup MCTC_WORKLOAD_CLUSTERS_COUNT=<NUMBER_WORKLOAD_CLUSTER>
+    ```
+
+3. Deploy the controller to the control plane cluster
+    ```sh
+    make deploy
+    ```
+4. Deploy the operator
+    ```sh
+    (export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs) && make build install run
```

Comment:
Does `make deploy` not deploy the operator on the cluster using the kustomize config? This is for running it locally no?

Reply:
Ooops copy paste error